packages/main/src/Popup.hbs

```diff
@@ -1,7 +1,7 @@
 <section
 	style="{{styles.root}}"
 	class="{{classes.root}}"
-	role="{{_role}}"
+	role={{_role}}
```

Comment:
The role here ends up in written PascalCase.
https://www.deque.com/blog/wai-aria-top-6-mistakes-to-avoid/#:~:text=Authors%20need%20to%20remember%20that,in%20lowercase